config.def.h

```diff
@@ -38,6 +38,7 @@ static const FilterFunc circle_grid_cross[] = {
 static const FilterSeq filter_default = FILTER_SEQ_FROM_ARRAY(circle_grid_cross);
 
 /* max time (in ms) allowed to go on without a redraw */
+/* -1 means wait for cursor movement */
```

Comment:
Not a fan of this since it was a stopgap solution and I will most likely end up removing this option altogether if there's a reliable way to detect change of content on the window below. But if that turns out to be not feasible then I won't have any problem merging this.

And there's also 0, which will do a redraw as soon as it can (pretty much guaranteed to blow up the cpu usage).